test/e2e/gcp.go

```diff
@@ -17,12 +22,17 @@ import (
 )
 
 type gcpTestHelper struct {
-	dnsService     *dns.Service
-	gcpCredentials string
-	gcpProjectId   string
+	dnsService      *dns.Service
+	gcpCredentials  string
+	gcpProjectId    string
+	providerOptions []string
 }
 
-func newGCPHelper(gcpCredentials, gcpProjectId string) (providerTestHelper, error) {
+func newGCPHelper(isOpenShiftCI bool, kubeClient client.Client) (providerTestHelper, error) {
+	gcpCredentials, gcpProjectId, err := prepareGCPConfigurations(isOpenShiftCI, kubeClient)
```

Comment:
Let's make `prepareConfigurations` a method of `gcpTestHelper`.

Reply:
Done, bring to member functions.